policy/modules/system/systemd.fc

```diff
@@ -4,6 +4,8 @@ HOME_DIR/\.config/systemd/user(/.*)?		gen_context(system_u:object_r:systemd_unit
 
 /etc/.*hostname.*	--	gen_context(system_u:object_r:hostname_etc_t,s0)
 /etc/machine-info	--	gen_context(system_u:object_r:hostname_etc_t,s0)
+/etc/systemd/.*\.conf	--	gen_context(system_u:object_r:systemd_conf_t,s0)
+/etc/systemd/.*\.conf\.d(/.*)?	gen_context(system_u:object_r:systemd_conf_t,s0)
```

Comment:
I guess this could be changed to `/etc/systemd/.*\.conf\.d(/.*\.conf)?`, but it doesn't matter much.

Reply:
Can be, although it may be another punch to clarity. Can there be only plain files?
```
/etc/systemd/.+\.conf	--	gen_context(system_u:object_r:systemd_conf_t,s0)
/etc/systemd/.+\.conf\.d	-d	gen_context(system_u:object_r:systemd_conf_t,s0)
/etc/systemd/.+\.conf\.d/.+\.conf	--	gen_context(system_u:object_r:systemd_conf_t,s0)
```

Reply:
This version is nice.

> Can there be only plain files?

Only `*.conf` is read. They don't have to be "plain", a symlink would be accepted too by the systemd code.

Reply:
OK, updated. The level of complexity now looks like an overkill.
I will merge it later tomorrow. thanks for the review.